.eslintrc.json

```diff
@@ -0,0 +1,14 @@
+{
+    "extends": ["prettier", "plugin:node/recommended"],
```

Comment:
As mentioned in the ticket, let's not extend any configurations. We'll derive our configuration based on popular configurations, but list everything in one place.

Reply:
I'm talking about `plugin:node/recommended`, not `prettier`.